### PR TITLE
Update manage-billing-notifications.md

### PR DESCRIPTION
**Pull request by emmasheridan-msft**
functionality listed is incorrect

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/commerce/billing-and-payments/manage-billing-notifications.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ Your organization's billing notifications are sent to the primary and alternate
 To change the primary and alternate email address of other admins in your organization, use the following steps.
 
 > [!NOTE]
-> Billing admins can change their own primary and alternate email addresses, but not for other admins.
+> Billing admins can only change their own alternate email address. They can not modify other admins' email addresses.
```
Comment (cmcatee-MSFT) [inline microsoft-365/commerce/billing-and-payments/manage-billing-notifications.md]: Please change to:

Billing admins can only change their own alternate email address. They can't change email addresses for other admins.

**Comment by yogkumgit** [comment]
@emmasheridan-msft please implement the suggested changes, in the event of no response we may have to close this PR. Thanks

**Comment by emmasheridan-msft** [comment]
@cmcatee-MSFT - do I need to do anything here?

**Comment by yogkumgit** [comment]
@emmasheridan-msft Yes. Please update the PR as per the feedback. Thanks

File: microsoft-365/commerce/billing-and-payments/manage-billing-notifications.md (hunk 1 above)
Comment (JohanFreelancer9) [inline microsoft-365/commerce/billing-and-payments/manage-billing-notifications.md]:
```suggestion
> Billing admins can only change their own alternate email address. They can't change email addresses for other admins.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@emmasheridan-msft, kindly commit the suggested change as per @cmcatee-MSFT's suggestion. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**